sentinelsat/sentinel.py

```diff
@@ -9,7 +9,7 @@
 import xml.etree.ElementTree as ET
 from collections import OrderedDict, defaultdict
 from contextlib import closing
-from datetime import date, datetime, timedelta
+import datetime
```

Comment:
This is just a matter of taste, but I prefer to use `import datetime as dt` since it's shorter and avoids ambiguity between the module and the `datetime` object in code.

Reply:
Everyone has their conventions. `dt` is for me usually a time difference, coming from physics. But perfectly fine here 😄 - so 👍 from me